src/components/SQForm/SQFormMultiValue.js

```diff
@@ -109,8 +110,41 @@ function SQFormMultiValue({
 
   const [inputValue, setInputValue] = React.useState('');
   const [customOptions, setCustomOptions] = React.useState([]);
+  const displayOptions = React.useMemo(() => [...children, ...customOptions], [
+    children,
+    customOptions
+  ]);
+  const previousInitialValue = usePrevious(initialValue);
 
-  const initialValue = [];
+  React.useEffect(() => {
+    // Don't run this effect every time displayOptions changes
+    // This also prevents infinite rerenders
+    if (JSON.stringify(previousInitialValue) === JSON.stringify(initialValue)) {
+      return;
+    }
+
+    const displayValues = displayOptions.reduce((acc, option) => {
+      acc.push(option.value);
+      return acc;
+    }, []);
```

Comment:
[nit] Maybe I am crazy, doesn't a map work better here or at least more specific? We start with one array and want to end up with another array of the same size.

Reply:
lol ya, you're right. that 5pm brain really doing my dirty on this PR. I remember my thinking was "I want to reduce this object to a single value, .reduce() will work!"